src/impl.cpp

```diff
@@ -720,7 +720,6 @@ std::ostream& operator<<(std::ostream& stream, const Manifold::Impl& impl) {
  * Write function.
  */
 Manifold Manifold::ReadOBJ(std::istream& stream) {
```

Comment:
Don't we have to guard the implementations the same way as the headers?

Reply:
Maybe pull master and check - I feel like this won't compile as is with `MANIFOLD_DEBUG=OFF`, or else I'm missing something about how these compilers work.

Reply:
Yes we do - there is an ifdef MANIFOLD_DEBUG guarding the whole block on line 685.  I forgot to remove it when I added the method internal versions in the previous commit, so I didn't need to re-add it here.

Reply:
Come to think of it - do all the CI tests have MANIFOLD_DEBUG=ON?